src/transformers/models/glm/configuration_glm.py

```diff
@@ -0,0 +1,137 @@
+# coding=utf-8
+# Copyright 2024 GLM & ZhipuAI team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""GLM model configuration"""
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+
+class GLMConfig(PretrainedConfig):
```

Comment:
There is still the issue with the camel casing!

Reply:
GLM is the name of our model, not Glm. Do we need to stick to camel case in this context as well?

Reply:
Yes, it's the same for LLaMa which we set to Llama!